src/main/java/edu/aau/groupc/canteenbackend/user/User.java

```diff
@@ -22,6 +26,9 @@
     private String username;
     private String password;
     private Type type;
+    @OneToMany(mappedBy = "user")
```

Comment:
Is `fetch = LAZY` not configured on purpose?

Reply:
It is the default, but i added it to keep the code consistent